app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
I have added comments about translation helper and singlequotes. Please fix them.